registration/admin.py

```diff
@@ -69,9 +71,16 @@ def save_model(self, request, obj, form, change):
 class ProfileAdmin(admin.ModelAdmin):
     """How should the profile be shown on the admin site."""
 
-    list_display = ['okuser', 'first_name', 'last_name', 'verified']
-    ordering = ['okuser']
+    list_display = [
+        'okuser',
+        'first_name',
+        'last_name',
+        'verified',
+        'created_at'
```

Comment:
```suggestion
        'created_at',
```
This helps keeping the diff small, when adding another item.